src/Dependencies/Collections/ImmutableSegmentedHashSet`1.cs

```diff
@@ -67,9 +118,8 @@ public ImmutableSegmentedHashSet<T> Add(T value)
             }
             else
             {
-                // TODO: Avoid the builder allocation
                 // TODO: Reuse all pages with no changes
```

Comment:
>// TODO: Reuse all pages with no changes [](http://example.com/codeflow?start=16&length=40)

Is this case covered by the else if clause above?

Reply:
No, this case is not covered at all.